.meta.toml

```diff
@@ -13,7 +13,7 @@ extra_lines = """
 [pyproject]
 codespell_skip = "*.min.js,*lock.yaml"
 codespell_ignores = "vew"
-dependencies_ignores = "['collective.folderishtypes', 'plone.app.caching', 'plone.app.multilingual', 'requests', 'responses', 'zest.releaser', 'zestreleaser.towncrier']"
```

Comment:
I think you would have to ignore `collective.folderishtypes`. It's in the code:

https://github.com/plone/plone.volto/blob/e5e357d4be45f8374fcffa676c462e832329a9d0/src/plone/volto/patches.py#L20

The dependency checker doesn't know it's inside the `try`.